src/core/utils/api-client/validators.ts

```diff
@@ -72,13 +72,15 @@ export class ValidatorsApiClient {
         address: string,
         blockchain: Blockchain,
         chainId: string,
+        appVersion?: string,
```

Comment:
Make sure on all blockchains this has been correctly.  Because `validatorId` is also optional, you need to be sure of that. Check on `Near`

Reply:
![image](https://user-images.githubusercontent.com/32679726/108739275-276f6a00-753d-11eb-86d0-57fc56683baf.png)